docs/faq.md

```diff
@@ -142,6 +142,12 @@ the `application.instanceLabelKey` value in the `argocd-cm`. We recommend that y
 
 See [#1482](https://github.com/argoproj/argo-cd/issues/1482).
 
+## How often does Argo CD sync applications ?
+
+The default sync interval is 3 minutes (180 seconds). 
```

Comment:
I think we're confusing sync and refresh here. Probably, the real FAQ should be:

"How often does Argo CD check for changes to my Git or Helm repository?"

And the answer should probably mention, that Argo CD will only sync upon detecting a change when autosync is enabled.

Reply:
@jannfis I updated the text according to your comment.